Final Project/Portfolio/css/styles.css

```diff
@@ -0,0 +1,207 @@
+/* 
+====== BRAND ====== 
+COLORS
+	blue #2C3E50
+	gray #ECF0F1
+	black #333333
+FONT
+	font-family: 'Raleway', Helvetica, Arial sans-serif;
+*/
+
+/* ====== GLOBAL ====== */
+html {
+	font-size: 16px;
+}
+* {
+	box-sizing: border-box;
+}
+body {
+	background-color: #ECF0F1;
+	font-family: 'Raleway', Helvetica, Arial sans-serif;
+}
+.content {
+	width: 70%;
+	margin: 0 auto;
+}
+
+/* ====== HEADER ====== */
+header {
```

Comment:
I would get a little more specific with these tags. Using a class like "page-header" and referring to it like that would allow you to use this tag again later.